lib/handlers/connect.js

```diff
@@ -81,6 +81,7 @@ function init (client, packet, done) {
 
   client.id = clientId || 'aedes_' + shortid()
   client.clean = packet.clean
+  client.version = packet.protocolVersion
```

Comment:
@gnought Looks good overall. I would be inclined towards changing the property from `version` to `protocolVersion` as it could easily be confused with the library version of the client rather than the protocol version, for those who don't read the documentation.

Reply:
@GavinDmello I think that if someone would like to get the library version it should get it from aedes and not a Client

Reply:
Oh. I meant, client.version sounds like the library version of the client which is connecting, which could be different for each each client. Don't think that should be a Aedes/broker level property or a property in our code at all.

Reply:
The concern from @GavinDmello comes to my mind when I go through the design process. It sounds like it's a library version but the point @robertsLando is correct, `Client` is a part of aedes module, not a library. `client.version` is designed to tell "this is MQTT 3.1 client, or MQTT 3.1.1 client or MQTT 5 client"
If we choose the name `client.protocolVersion`, it it weird that telling "hey what protocolVersion are you" where the client is not a packet.

Reply:
That was a suggestion. Happy for this to go ahead.